{{cookiecutter.repo_name}}/.pre-commit-config.yaml

```diff
@@ -32,8 +32,8 @@ repos:
   hooks:
       - id: flake8
         types: [python]
-        exclude: setup.py
```

Comment:
sorry Chris we've probably already spoken about this but just want to be absolutely certain that removing setup.py from the flake8 exclude list won't cause us grief?

Reply:
We did speak about this and the general idea was that we should check as many files as possible with the tests, so that files are as compliant as possible. However, previous versions of the template ignore setup.py so in the interest of compatibility it might be better to continue excluding it